dashboard/client/src/pages/log/Logs.tsx

```diff
@@ -77,6 +78,7 @@ const useLogs = ({ theme, newIA }: LogsProps) => {
     } else {
       setOrigin(undefined);
     }
+    setDownloadUrl(getLogDownloadUrl(url));
```

Comment:
nit: think this should be inside L85
```typescript
        if (res) {
          setLogs(res);
          setDownloadUrl(getLogDownloadUrl(url));
```

otherwise the download button shows regardless of logs actually existing

Reply:
I think it's fine for that case. Then the download will be empty. No need to wait for the whole logs to download before showing the download link.